src/main/webapp/js/adminHome.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * This function generates HTML text for a row containing instructor's information
+ * and status of the action.
+ * 
+ * @param {String} shortName
+ * @param {String} name
+ * @param {String} email
+ * @param {String} institution
+ * @param isSuccess
+ * @param {String} status
+ * @returns {String} a HTML row of action result table
+ */
+function createRowForResultTable(shortName, name, email, institution, isSuccess, status) {
+    var result = "<td>" + shortName + "</td>";
+    result += "<td>" + name + "</td>";
+    result += "<td>" + email + "</td>";
+    result += "<td>" + institution + "</td>";
+    if (isSuccess) {
+        result += "<td>Success</td>";
+    } else {
+        result += "<td>Fail</td>";
+    }
+    result += "<td>" + status + "</td>";
+    if (isSuccess) {
+        result = "<tr class=\"success\">" + result + "</tr>";
+    } else {
+        result = "<tr class=\"danger\">" + result + "</tr>";
+    }
+    return result;
+}
+
+var paramsCounter = 0;
+var paramsList = [];    // list of parameter strings that will be sent via ajax
+var instructorDetailsList = [];
+var isInputFromFirstPanel = false;
+
+/**
+ * This function disables the form to prevent the user from editing it
+ * while the action is being processed.
```

Comment:
Omit details about when the function is used. We can only control what it does; we can't control when it will be used.